databind/src/main/java/gov/nist/secauto/metaschema/databind/codegen/impl/AnnotationGenerator.java

```diff
@@ -197,7 +197,8 @@ private static void applyLetAssignments(
       // TODO: Support remarks
       // MarkupMultiline remarks = let.getRemarks();
       // if (remarks != null) {
-      // constraintAnnotation.addMember("remarks", "$S", remarks.toMarkdown());
+      // constraintAnnotation.addMember("remarks", "$S",
+      // remarks.toMarkdown());
       // }
```

Comment:
Non-blocking: do we need to still track this feature of m-j? And if so, should we clean up the code and/or link to an issue instead of leaving it commented out?

Reply:
Added https://github.com/metaschema-framework/metaschema-java/issues/169